CHANGES.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+* Changes
+  * Added support for Cesium rectangles via the olcs.polygon_kind property.
```

Comment:
The description could be more detailed: `Add support for drawing rectangles according to the longitude and latitude curves instead of straight lines. This functionality can be activated by setting the olcs.polygon_kind property to 'rectangle' on the OpenLayers geometry.`